contrib/format-daffodil/src/main/java/org/apache/drill/exec/store/daffodil/DaffodilBatchReader.java

```diff
@@ -64,64 +69,97 @@ public DaffodilBatchReader (DaffodilReaderConfig readerConfig, EasySubScan scan,
     this.validationMode = formatConfig.getValidationMode();
 
     //
-    // FIXME: Next, a MIRACLE occurs.
+    // FIXME: Where is this config file to be found? And, what is its syntax?
     //
-    // We get the dfdlSchemaURI filled in from the query, or a default config location
-    // We get the rootName (or null if not supplied) from the query, or a default config location
-    // We get the rootNamespace (or null if not supplied) from the query, or a default config location
-    // We get the validationMode (true/false) filled in from the query or a default config location
-    // We get the dataInputURI filled in from the query, or from a default config location
-    //
-    // For a first cut, let's just fake it. :-)
+    // FIXME: How do I arrange for these same things to be overriddable in the query
+    //   or has that already happened before we get these things?
```

Comment:
This has already happened.  If you need any variables to be configurable by the user, just add them to `DaffodilFormatConfig`.

Reply:
Great.  ... but I still don't know where does Drill look for this config file, what is its naming convention and what is its syntax?